src/content/docs/dropins/product-details/pdp-containers.mdx

```diff
@@ -141,6 +128,17 @@ Triggered when the Add to Cart button is clicked. Receives an object with `sku`,
 onAddToCart: (values) => void;
 ```
 
+## zoomType
+
+This property can take one of two string values: `zoom` or `overlay`. It is used to enhance product image viewing by providing zoom or overlay functionality:
+
+- `zoom`: Provides a close-up view of product details.
+- `overlay`: Provides a larger, distraction-free view of a product image.
+
+## closeButton
+
+This property is a boolean that controls the visibility of a close button on modals and overlays. If `true`, the close button is shown; if `false` or not provided, the close button is not shown.
```

Comment:
This property can only be used in combination with **zoomType**. 
If **zoomType** is set to `zoom` and **closeButton** is set to `true` → then this is the resulting experience - https://experience-qa.adobe.net/solutions/magento-datalake-storefront-pdp/assets/qa20240912080555/standalone/product.html?sku=VSW12. Image is zoomed inside its bounds

If **zoomType** is set to `zoom` and **closeButton** is set to `false` (or unspecified) → Image is zoomed in its bounds, but the close button is not displayed.

If **zoomType** is set to `overlay` (or unspecified) → the image is opened in a full screen modal, as it can be seen here: https://experience-qa.adobe.net/solutions/magento-datalake-storefront-pdp/assets/qa20240912080555/standalone/product.html?sku=VA11-GO-NA.

Reply:
Default value for `zoomType` is `overlay`.
Default value for `closeButton` is `false`.